LearningMissionSimulation/PlaygroundMher.cs

```diff
@@ -156,14 +185,35 @@ List<Module> GetModuleList()
 
         public void CreateClassrooms(int classroomCount)
         {
-            throw new NotImplementedException();
+            Guid moduleId = moduleIdList[AttributeGenerator.random.Next(0, moduleIdList.Count)];
+            Module module;
+            moduleDictionary.TryGetValue(moduleId, out module);
+            int i = 0;
+            while (i < classroomCount)
+            {
+                Classroom classroom = ObjectGenerator.GenerateClassroom(module);
+                classroomList.Add(classroom);
+                i++;
+            }
         }
 
-       
+        public void AssignInstructorsToClassrooms()
+        {
+            foreach (var classroom in classroomList)
+            {
+                if (moduleInstructorDictionary.ContainsKey(classroom.Module.Id))
```

Comment:
Add an **else** block to handle the case when  moduleInstructorDictionary does not contain the module's id that we need for the classroom. That means that there is not instructor that can teach this module. You can just report in this case something like "Failed to find instructor for classroom {}"